AppCheckCore/Sources/Public/AppCheckCore/GACAppCheckErrors.h

```diff
@@ -55,5 +55,6 @@ typedef NS_ENUM(NSInteger, GACAppCheckMessageCode) {
 
   // App Attest Provider
   GACLoggerAppCheckMessageCodeAppAttestNotSupported = 7001,
-  GACLoggerAppCheckMessageCodeAttestationRejected = 7002
+  GACLoggerAppCheckMessageCodeAttestationRejected = 7002,
+  GACLoggerAppCheckMessageCodeAssertionRejected = 7003
```

Comment:
todo: test if this is a breaking change

Reply:
It's technically a breaking change for clients switching over the values but a code search yields no usage.